runners/genlayer-py-std/src/genlayer/gl/vm.py

```
@@ -0,0 +1,174 @@
+import collections.abc
+
+from genlayer.py.types import Lazy
+from ._internal import _lazy_api
+import genlayer.py.calldata as calldata
+import genlayer.gl._internal.gl_call as gl_call
+
+from ._internal.result_codes import ResultCode
+
+
+@dataclasses.dataclass
+class Return[T: calldata.Decoded]:
```

Pylint (3.3.7)

[error] Parsing failed: 'invalid syntax (genlayer.gl.vm, line 25)'

(E0001)